lib/minfraud/transaction.rb

```diff
@@ -100,7 +100,7 @@ def validate_attributes
     # @return [nil, TransactionError]
     def validate_string(attr_name)
       attribute = self.send(attr_name)
-      unless attribute.instance_of?(String)
+      if attribute && !attribute.instance_of?(String)
         raise TransactionError, "Transaction.#{attr_name} must me a string"
```

Comment:
I know it is not from your change, but can you fix the typo in the error message?